### PR TITLE
182 upload a video

### LINKED ISSUE #182: Upload a Video

### PR DESCRIPTION
**Pull request by ichung08**
Created new guide for upload a video asset

Resolves #182, #184

### CODE REVIEW

**Comment by adamsoffer** [comment]
@ichung08 I'll submit my review soon. Heads up, I managed to get tabbed pages working with proper url routing and table of contents so we can better present dashboard + api instructions within a single guide. I pushed up my implementation to your working branch. Have a look at [this commit](https://github.com/livepeer/livepeer-studio-docs/pull/186/commits/fed1bc13d3e89cadbf62541d5a06cb271f481f89) to see how I did it so you can implement it on the other guides as well. 
Screen Shot 2022-08-26 at 5 23 41 PM

**Comment by ichung08** [comment]
Sounds good! I'll take a look and update my branch

**Review by adamsoffer — CHANGES_REQUESTED** [review]
- [x] Can we remove the step 1 instructing the user to log into the dashboard? I think we can give the developer the benefit of the doubt on this one.
- [x] Rename "Technical preparation" to "Prerequisites"
- [x]  I noticed some codeblocks are labeled "javascript" and some are labeled "Typescript". Let's be more consistent here and rename all these code blocks `Node`. 
- [x] Add a `curl` example to each code block.
- [x] Let's refer to the API as simply "the Livepeer API" not "the Video-on-Demand (VOD) API"
- [ ] Use [this tool called screely](https://www.screely.com/) for all screenshot images to make them look more presentable. When using the tool use a hex background of #151718.
- [ ] Screenshot images right now appear too large. Install [this plugin](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-ideal-image) for image size formatting and better performance.

**Comment by ichung08** [comment]
> Use [this tool called screely](https://www.screely.com/) for all screenshot images to make them look more presentable. When using the tool use a hex background of #151718.

Is there a certain style you would like the screenshot images to appear? As in, preferences regarding window border, amount of vertical and horizontal padding, etc

File: src/components/TabbedPages/index.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ function TabbedPages({ pages }) {
                 activePage === slug
                   ? "var(--ifm-tabs-color-active)"
                   : "var(--ifm-tabs-color)",
+              fontSize: 25,
```
Comment (adamsoffer) [inline src/components/TabbedPages/index.tsx:25]: @ichung08 Let's reduce the tab font size to 20px.

Comment (adamsoffer) [inline src/components/TabbedPages/index.tsx:25]: @ichung08 let's reduce the tab font size to 20px

**Review by pglowacky — APPROVED** [review]
Looks good!

File: docs/04-guides/on-demand/upload-a-video-asset/01-dashboard.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+sidebar_class_name: tabbed-page
+hide_title: true
+slug: dashboard
+title: Dashboard
+metaTitle: Upload a video Asset
```
Comment (adamsoffer) [inline docs/04-guides/on-demand/upload-a-video-asset/01-dashboard.mdx:6]: A couple frontmatter fixes:
- [ ]  We can remove the `metaTitle` frontmatter field. This isn't actually a field docusaurus recognizes.
- [ ] Let's add `pagination_next: guides/update-a-video-asset` so it links to an appropriate next guide at the end of the doc. Right now it links to the api tab.

File: docs/04-guides/on-demand/upload-a-video-asset/02-api.mdx [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+import Tabs from "./_tabs.mdx";
+import CodeBlock from "@theme/CodeBlock";
+import CodeBlockTabs from "@site/src/components/CodeBlockTabs";
+import CodeBlockTabItem from "@site/src/components/CodeBlockTabItem";
+
+<Tabs />
+
+The Livepeer API allows you to send video files to Livepeer and get
+them ready for optimized playback. Those videos can be provided either by you
```
Comment (adamsoffer) [inline docs/04-guides/on-demand/upload-a-video-asset/02-api.mdx:16]: "~Those~ Videos can be provided..."

**Review by adamsoffer — CHANGES_REQUESTED** [review]

File: docs/04-guides/on-demand/upload-a-video-asset/01-dashboard.mdx (hunk 2 above)
Comment (ichung08) [reply docs/04-guides/on-demand/upload-a-video-asset/01-dashboard.mdx:6]: I'll add pagination_next for now but I think we'll have to adjust this later as the other guides for on demand get added

**Review by adamsoffer — APPROVED** [review]

**Pull request merged**